### DISCUSSION TITLE
pc端更新4.12之后没有pac模式了？

### DISCUSSION THREAD

**Discussion by 0f7b90** [v2fly/v2ray-core#776] [Q&A]
并不想让所有网站都走代理，很影响日常浏览的速度，但用旧版本覆盖之后原来的自定义pac列表就清空了。想问下有没有办法能退回到有pac模式和列表的版本，或者在当前最新版本中找回原有的pac模式？十分感谢

**Comment by Loyalsoldier** [comment]
v2rayN 提问请到 https://github.com/2dust/v2rayN/issues

**Comment by Cwek** [comment]
v2ray核心没实现PAC的支持，那是外围壳（类似v2rayN等）的问题。
